plugins/performance-lab/includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -211,7 +211,24 @@ function perflab_aao_get_autoloaded_options_table(): string {
 function perflab_aao_get_disabled_autoloaded_options_table(): string {
 	$disabled_options = get_option( 'perflab_aao_disabled_options', array() );
 
-	if ( ! is_array( $disabled_options ) ) {
```

Comment:
This type check should remain somewhere because the option could be populated with anything. If removing it, then at least add an `(array)` cast to the `get_option()` call.

Reply:
Added in https://github.com/WordPress/performance/pull/1374/commits/cd7458bd51a5199a763cef865a05d300987897d5